test/system/super_scaffolding_test.rb

```diff
@@ -212,4 +212,99 @@ def setup
       assert page.has_content? "One and Two"
     end
   end
+
+  if defined?(PartialTest)
+    test "super scaffolded partials function properly" do
+      display_details = @@test_devices[:macbook_pro_15_inch]
+      resize_for(display_details)
+
+      login_as(@jane, scope: :user)
+      visit account_team_path(@jane.current_team)
+
+      click_on "Add New Partial Test"
+
+      # Text Field partial
+      fill_in "Text Field Test", with: "Test Text"
+      # Boolean Button partial
+      choose "No"
```

Comment:
I changed this from `click_on` to `choose` since booleans were changed to radio buttons in [this PR in bullet_train-super_scaffolding](https://github.com/bullet-train-co/bullet_train-super_scaffolding/pull/23)